scripts/release.py

```
@@ -46,12 +238,50 @@ def print_help():
         print_help()
         sys.exit(0)
 
+    # Return command and options
+    return args.command, {
+        "version_type": args.type,
+        "prerelease": args.pre,
+        "dry_run": args.dry_run,
+    }
+
+
+def main():
```

Ruff (0.8.2)

`main` is too complex (24 > 15)

(C901)